grails-app/domain/com/causecode/fileuploader/UFile.groovy

```diff
@@ -65,6 +66,7 @@ class UFile implements Serializable {
         dateCreated bindable: false
         lastUpdated bindable: false
         envName bindable: false
+        containerName nullable: true
```

Comment:
should we also add min-length validations to it

Reply:
Added